packages/matter-node.js/test/crypto/CryptoTest.ts

```diff
@@ -91,5 +85,18 @@ describe("Crypto", () => {
 
             cryptoNode.verify(key, ENCRYPTED_DATA, cryptoNode.sign(key, ENCRYPTED_DATA));
         });
+
+        // Too slow to leave enabled by default but useful for confirming key variance doesn't cause failure
+        // it("creates a large volume of private keys", () => {
+        //     for (let i = 0; i < 100000; i++) {
```

Comment:
maybe just lower the iterations to 10000 or such?

Reply:
Still too slow for my taste (7 seconds at 10k iterations)...  1k would probably be OK but isn't much of a test